core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -539,4 +539,13 @@ class EMConfig {
      */
     @Cfg("QWN0aXZhdGUgdGhlIFVuaWNvcm4gTW9kZQ==")
     var e_u1f984 = false
+
```

Comment:
add a bit more description in the `@Cfg` annotations

Reply:
I think that's the unicorn. I must've screwed something up during the merge.

Reply:
issue with GitHub: it is showing only the snippet above the comment, whereas I meant the 2 new `@Cfg` after that below

Reply:
I've expanded the descriptions of those a bit. I'll expand them more, if needed. I'll push the changes in a minute.